plugins/websocket_server/WebsocketServer.cc

```diff
@@ -806,7 +806,7 @@ void WebsocketServer::OnMessage(int _socketId, const std::string &_msg)
     bool result;
     unsigned int timeout = 2000;
 
-    bool executed = this->node.Request("/gazebo/worlds",
+    bool executed = this->node.Request("/sim/worlds",
```

Comment:
I still see it here

https://github.com/gazebosim/gz-launch/blob/cf8e781b380e1109c382cc2026dcfd29acfc27ec/plugins/websocket_server/WebsocketServer.cc#L809-L810

Reply:
Oops: https://github.com/gazebosim/gz-launch/pull/176/commits/db670067ca0cb706c1cc18383d82c220e05011cc